include/taskolib/Sequence.h

```diff
@@ -69,6 +69,21 @@ namespace task {
  * // insert an action step at index 1
  * seq.insert(seq.begin() + 1, Step{Step::type_action});
  * \endcode
+ *
+ * \section Step setup script
+ *
+ * Each step can have a setup script that is executed before the intrinsic script is
+ * called (invoked with Step::set_script(const std::string&)). You can call it with
```

Comment:
I find the first sentence slightly misleading. "Each step can have a setup script" implies "each step can have a *different* setup script` for me. I also find the "invoked" comment unnecessary. Maybe something like this?

> The sequence contains a common setup script that is shared by all of its steps. It is called automatically before the execution of each step's script, just after executing the `lua_step_setup` function from the context. This step setup script can be set with `set_step_setup_script()`. It is typically used like a small library for defining common Lua functions or constants.

Reply:
You have a point there. I had not even thought about this script being tied to a scripting language or not. If we ever get a second one - say, Python - we'll need to decide how to handle these things. Maybe we'll need one Lua setup script and a Python setup script.

Reply:
Yes, this should be discussed when the next scripting language is included into the lib.